src/l500/l500-depth.h

```diff
@@ -178,6 +178,12 @@ namespace librealsense
             intrinsics.fy = intrinsic_params.pinhole_cam_model.ipm.focal_length.y;
             intrinsics.ppx = intrinsic_params.pinhole_cam_model.ipm.principal_point.x;
             intrinsics.ppy = intrinsic_params.pinhole_cam_model.ipm.principal_point.y;
+            intrinsics.coeffs[0] = intrinsic_params.pinhole_cam_model.distort.radial_k1;
+            intrinsics.coeffs[1] = intrinsic_params.pinhole_cam_model.distort.radial_k2;
+            intrinsics.coeffs[2] = intrinsic_params.pinhole_cam_model.distort.radial_k3;
+            intrinsics.coeffs[3] = intrinsic_params.pinhole_cam_model.distort.tangential_p1;
+            intrinsics.coeffs[4] = intrinsic_params.pinhole_cam_model.distort.tangential_p2;
+            intrinsics.model = RS2_DISTORTION_NONE;
```

Comment:
Thanks for the quick response, the parameters should be reordered and to be aligned with the [struct's layout](https://github.com/IntelRealSense/librealsense/blob/ac/src/l500/l500-private.h#L226-L230),
similar to https://github.com/IntelRealSense/librealsense/blob/ac/src/l500/l500-color.cpp#L202-L206:
```cpp
                if( model.distort.radial_k1 || model.distort.radial_k2 || model.distort.tangential_p1 || model.distort.tangential_p2 || model.distort.radial_k3 )
                {
                    intrinsics.coeffs[0] = model.distort.radial_k1;
                    intrinsics.coeffs[1] = model.distort.radial_k2;
                    intrinsics.coeffs[2] = model.distort.tangential_p1;
                    intrinsics.coeffs[3] = model.distort.tangential_p2;
                    intrinsics.coeffs[4] = model.distort.radial_k3;

                    intrinsics.model = RS2_DISTORTION_INVERSE_BROWN_CONRADY;
                }
```

Reply:
Oh that is a bit confusing. Any reason why the coeffs are handed out in an array rather than the distortion struct directly? Would make things a lot clearer.

Reply:
The parameters are unique per distortion model in use, so apparently the names cannot be hard-coded/reused.
As for the Brown-Conrady distortion and its coefficients - this convention used throughout the SDK for all sensor's that are modeled after it